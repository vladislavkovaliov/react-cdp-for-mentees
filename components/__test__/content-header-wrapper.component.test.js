import React from 'react';
import { ContentHeaderWrapper } from '../content-header-wrapper.component';

import shallow from 'enzyme/shallow';

describe('<ContentHeaderWrapper />', () => {
  test('Shapshot <ContentHeaderWrapper />', () => {
    const wrapper = shallow(
      <ContentHeaderWrapper>
        <div>hello</div>
      </ContentHeaderWrapper>
    );

    expect(wrapper).toMatchSnapshot();
  });
});
