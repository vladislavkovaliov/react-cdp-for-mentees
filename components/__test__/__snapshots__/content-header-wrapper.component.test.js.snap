// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ContentHeaderWrapper /> Shapshot <ContentHeaderWrapper /> 1`] = `
<div
  className="content__wrapper-header"
>
  <div
    className="content__header"
  >
    <div>
      hello
    </div>
  </div>
</div>
`;
